content/docs/user-guide/scripting/script-canvas/get-started/_index.md

```diff
@@ -5,4 +5,4 @@ description: Tour Script Canvas to start learning the basics of using the visual
 weight: 100
 ---
 
-This guide shows you the basic concepts of Script Canvas. It provides you with a tour of the Script Canvas Editor to help you get started using Script Canvas. We also provide you a glossary of terms for you to refer back to as you learn your way around.
+This guide shows you the basic concepts of Script Canvas. It provides you with a tour of the Script Canvas Editor to help you get started. We also provide you with a glossary of terms to refer back to as you learn your way around.
```

Comment:
I've been removing "we" wherever I find it. In this context it seems odd to refer to what the guide provides ("it") and then shift to referring to what "we" provide.

```suggestion
This guide shows you the basic concepts of Script Canvas. It provides you with a tour of the Script Canvas Editor to help you get started and a glossary of terms to refer to as you learn your way around.
```